app/ui-flextab/client/tabs/userActions.js

```diff
@@ -111,6 +111,24 @@ export const getActions = ({ user, directActions, hideAdminControls }) => {
 			},
 		},
 
+		{
+			icon: 'articles',
```

Comment:
UX question, should we show this button is user's account is not on ghost?

Reply:
If we do this, and store and maintain a state in user objects > ghostAccountExists,,.. then this can simply be a link and the corresponding meteor method is not required.